.github/workflows/terraform.yml

```diff
@@ -0,0 +1,81 @@
+# Some of this logic was stolen from https://www.freecodecamp.org/news/a-lightweight-tool-agnostic-ci-cd-flow-with-github-actions/
```

Comment:
@ddriddle, I should have asked you not to worry about reviewing the GitHub actions. The file here is a minimally-viable version borrowed from other Terraform projects, and as such does not represent anything new to this repo, nor does it represent what I hope to attain as a gold standard for GitHub Actions for Terraform modules.

This is one of the areas that I hope to work on with you in the future after Shibboleth has died down. To that end, I'm going to ignore the comments on the CI pieces, with the thought that we'll address them together soon, add them to SDG's templates, and disperse them broadly across all our repos so that we can stick a fork in Drone.